tests/test_tags.py

```diff
@@ -256,3 +261,17 @@ def test_urgent_count(hlwm, x11):
 
     # since one of them gets focused, 4 urgent clients remain
     assert int(hlwm.get_attr('tags.focus.urgent_count')) == 4
+
+
+def test_rename_different_tags(hlwm, hc_idle):
```

Comment:
I think `test_rename_multiple_tags` is a better name.

Reply:
True, changed that. Thanks!